### PR TITLE
Fix deployment by removing the requirement to install `slackbot`, sin…

### PR DESCRIPTION
**Pull request by kjkoster**
…ce that was merged into the backend. Documentation updates.

### CODE REVIEW

File: docker-compose-rag-doll-only.yml
Comment (anjarakvo) [inline docker-compose-rag-doll-only.yml:1]: To make the API work, it seems you also need to add the backend service to this docker-compose file.

**Review by anjarakvo — CHANGES_REQUESTED** [review]

File: docker-compose-rag-doll-only.yml
Comment (kjkoster) [reply docker-compose-rag-doll-only.yml:1]: Will do.

**Comment by kjkoster** [comment]
@anjarakvo Keep in mind that this change means that your deployed `.env` needs to change quite a bit. You can also remove the firewall ports that you opened for the Slack and Twilio bots. These no longer exist, so the ports may be closed.

@dedenbangkit I notice there is a `/dev` at the end of the database URL. Is that proper for Kubernetes also, or should that be a different URL?

**Comment by dedenbangkit** [comment]
> @anjarakvo Keep in mind that this change means that your deployed `.env` needs to change quite a bit. You can also remove the firewall ports that you opened for the Slack and Twilio bots. These no longer exist, so the ports may be closed.
> 
> @dedenbangkit I notice there is a `/dev` at the end of the database URL. Is that proper for Kubernetes also, or should that be a different URL?

@anjarakvo stored the values of all the environment variables somewhere else (usually in secret repository). Anything related to the Docker Compose values will be overridden by the .yaml file that he has. There is no need to worry about the environment values in Docker Compose. Since this is a different standard deployment, I don't know where Anjar stores the template.yml file. CMIIW @anjarakvo

**Comment by kjkoster** [comment]
> @anjarakvo stored the values of all the environment variables somewhere else (usually in secret repository). Anything related to the Docker Compose values will be overridden by the .yaml file that he has. There is no need to worry about the environment values in Docker Compose. Since this is a different standard deployment, I don't know where Anjar stores the template.yml file. CMIIW @anjarakvo

In that case, should we not remove the environment variables from the Compose file? No point in having them if they get ignored.

**Comment by dedenbangkit** [comment]
> > @anjarakvo stored the values of all the environment variables somewhere else (usually in secret repository). Anything related to the Docker Compose values will be overridden by the .yaml file that he has. There is no need to worry about the environment values in Docker Compose. Since this is a different standard deployment, I don't know where Anjar stores the template.yml file. CMIIW @anjarakvo
> 
> In that case, should we not remove the environment variables from the Compose file? No point in having them if they get ignored.

But it will be required for the CI pipeline if there's any test. Although we use environment variables in Docker Compose, the environment will not be available after the Docker image is built.

**Review by dedenbangkit — APPROVED** [review]

**Review by anjarakvo — APPROVED** [review]

**Pull request merged**